src/Aquifer.Jobs/Program.cs

```diff
@@ -1,34 +1,47 @@
 using Aquifer.Common.Clients;
 using Aquifer.Common.Clients.Http.IpAddressLookup;
+using Aquifer.Common.Jobs;
 using Aquifer.Common.Services;
 using Aquifer.Data;
 using Aquifer.Jobs.Clients;
 using Aquifer.Jobs.Configuration;
+using Aquifer.Jobs.Services;
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
 
-var host = new HostBuilder().ConfigureFunctionsWorkerDefaults()
+var host = new HostBuilder()
+    .ConfigureFunctionsWorkerDefaults()
     .ConfigureServices((context, services) =>
     {
         services.AddOptions<ConfigurationOptions>().Bind(context.Configuration);
 
-        var connectionString = context.Configuration.GetConnectionString("BiblioNexusDb");
+        var configuration = context.Configuration.Get<ConfigurationOptions>()
+            ?? throw new InvalidOperationException($"Unable to bind {nameof(ConfigurationOptions)}.");
+
+        services.AddDbContext<AquiferDbContext>(options => options
+            .UseSqlServer(configuration.ConnectionStrings.BiblioNexusDb, providerOptions => providerOptions.EnableRetryOnFailure(3))
+            .EnableSensitiveDataLogging(sensitiveDataLoggingEnabled: configuration.IsDevelopment)
+            .UseQueryTrackingBehavior(QueryTrackingBehavior.NoTracking));
```

Comment:
I missed adding EF `NoTracking` for Aquifer.Jobs previously.  All but one existing query is read-only and I had already added `AsTracking()` to the write query.